libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKUpgradeManager.java

```diff
@@ -115,7 +109,9 @@ protected synchronized void upgradeAccMgr() {
             	upgradeTo2Dot2();
             }
         } catch (NumberFormatException e) {
-        	Log.e("UpgradeManager:upgradeAccMgr", "Failed to parse installed version.");
+            SalesforceLogger.getLogger(SalesforceSDKManager.SF_SDK_COMPONENT_NAME,
```

Comment:
Can we make that shorter?
Maybe SalesforceSDKManager could have a static method getLogger(String componentName) and the logger could have a e method (for errors), and you could import static SF_SDK_COMPONENT_NAME, so it would look like
```java
SalesforceSDKManager.getLogger(SF_SDK_COMPONENT_NAME).e(TAG, "Failed ...")
```